parquet-hadoop/src/main/java/parquet/hadoop/MemoryManager.java

```diff
@@ -55,7 +55,7 @@ public MemoryManager(float ratio, long minAllocation) {
     minMemoryAllocation = minAllocation;
     totalMemoryPool = Math.round(ManagementFactory.getMemoryMXBean().getHeapMemoryUsage().getMax
         () * ratio);
-    LOG.debug(String.format("Allocated total memory pool is: %,d", totalMemoryPool));
+    LOG.info(String.format("Allocated total memory pool is: %,d", totalMemoryPool));
```

Comment:
I'm not sure we want this to show up in the case where the memory manager does not do anything special (scale == 1).
It will add noise for user that have only one writer.